capstone/src/test/java/com/google/univiz/scorecard/ScorecardConverterTest.java

```diff
@@ -0,0 +1,92 @@
+package com.google.univiz.scorecard;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.common.base.Converter;
+import com.google.univiz.CarnegieSizeDegree;
+import com.google.univiz.CollegeData;
+import com.google.univiz.CollegeId;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ScorecardConverterTest {
+
+  private static final int COLLEGE_ID = 193900;
+  private static final String NAME = "New York University";
+  private static final String CITY = "New York";
+  private static final int FLAG_MAIN_CAMPUS = 1;
+  private static final double LATITUDE = 40.729452;
+  private static final double LONGITUDE = -73.997264;
+  private static final int CARNEGIE_SIZE_DEGREE = 17;
+  private static final double ADMISSION_RATE = 0.1999;
+  private static final double AVG_SAT = 1419.0;
+  private static final int NUM_OF_UNDERGRADS = 26339;
+  private static final int AVG_COST = 69830;
+  private static final double RATIO_OF_MEN = 0.4253;
+  private static final double RATIO_OF_WOMEN = 0.5747;
+
```

Comment:
```suggestion

private final ScorecardConverter converter = new ScorecardConverter();

```

You can move the converter to a common field and then remove the instantiation from both tests...